weasyl/submission.py

```diff
@@ -709,10 +709,23 @@ def twitter_card(submitid):
         content = "[This submission has no description.]"
 
     ret = {
-        'url': d.absolutify_url('/submission/%s/%s' % (submitid, text.slug_for(title))),
-        'description': content,
+        'url': d.absolutify_url(
+            '/submission/%s/%s' % (submitid, text.slug_for(title))),
     }
 
+    if twitter:
+        ret['creator'] = '@%s' % (twitter.lstrip('@'),)
+        ret['title'] = title
+    else:
+        ret['title'] = '%s by %s' % (title, full_name)
+
+    if rating > 20:
```

Comment:
The future-proof, non-magic-number way to do this would be using the `Rating.minimum_age` attribute from `libweasyl.ratings`. Something like `if ratings.CODE_MAP[rating].minimum_age >= 18:`